src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5593,6 +5607,40 @@ public override IOperation VisitOmittedArgument(IOmittedArgumentOperation operat
             return new OmittedArgumentExpression(semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
+        internal override IOperation VisitPlaceholder(IPlaceholderOperation operation, int? captureIdForResult)
```

Comment:
Why did this method move?

Reply:
> Why did this method move?

The region below contains methods that haven't been finalized. Once that happens, we move the method out of it.